addons/project/views/project_views.xml

```diff
@@ -1114,16 +1127,23 @@
                                     <field name="legend_blocked" invisible="1"/>
                                     <field name="project_id" invisible="1"/>
                                     <field name="is_closed" invisible="1"/>
-                                    <field name="sequence" widget="handle"/>
-                                    <field name="name"/>
+                                    <field name="allow_subtasks" invisible="1" />
```

Comment:
I would place it just before the `name` field as we need it because of `name_with_subtask_count`